src/combinatorics/simple_graph/connectivity.lean

```diff
@@ -482,13 +556,21 @@ lemma support_drop_until_subset {u v w : V} (p : G.walk v w) (h : u ∈ p.suppor
   (p.drop_until u h).support ⊆ p.support :=
 λ x hx, by { rw [← take_spec p h, mem_support_append_iff], exact or.inr hx }
 
+lemma darts_take_until_subset {u v w : V} (p : G.walk v w) (h : u ∈ p.support) :
```

Comment:
Would it make sense to interface this with a `subwalk` predicate? I think not, because the best definition I can think of for `p.subwalk q` is `p.darts ⊆ q.darts`.